galaxy.yml

```diff
@@ -8,7 +8,7 @@ namespace: redpanda
 name: cluster
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 0.4.11
+version: 0.4.12
```

Comment:
WDYM you can just update the galaxy version in the main PR!? My brain swears it is only legal in a second, follow on PR urgently pushed after it was forgotten in the first PR 😂